themes/digital.gov/layouts/partials/core/featured-resource.html

```diff
@@ -0,0 +1,33 @@
+{{- $resourcePath := .resourcePath -}}
+{{- $kicker := .kicker -}}
+{{- if $resourcePath -}}
+  {{- $resource := .Site.GetPage $resourcePath -}}
+  {{- if $resource -}}
+
+    <div class="dg-featured-resource">
+      <a
+        href="{{- $resource.Permalink | absURL -}}"
```

Comment:
Curious as to why this is an `absURL`, is this for links _outside_ of DigitalGov as well?